code/modules/mod/modules/_module.dm

```diff
@@ -47,6 +47,7 @@
 	var/allowed_inactive = FALSE
 	/// Timer for the cooldown
 	COOLDOWN_DECLARE(cooldown_timer)
+	cooldown_timer = 2
```

Comment:
```suggestion
	COOLDOWN_DECLARE(cooldown_timer = 2)
```

This also doesn't fix it, because from what I see nothing ever uses COOLDOWN_START()